tests/test_sdk.py

```diff
@@ -219,3 +219,9 @@ def test_job():
                 continue
             assert log in all_text_sent_to_server
         assert str(test_obj) in all_text_sent_to_server
+
+    def test_metric(self):
```

Comment:
I think we can write a better test than this. Emit a few different metrics names, with a few different values for each. In addition, if you look at previous tests, you can use the global server_sdk_messages to assert that the correct messages were sent.